src/lib/index.ts

```diff
@@ -2,6 +2,17 @@ import fetch from 'node-fetch';
 import path from 'path';
 import execSh from 'exec-sh';
 import { specmaticJarPathLocal } from '../config';
+import fs from 'fs';
+
+
+
+export const setSpecmaticEnvVariable = (name: string, value: string, fileName: string) => {
```

Comment:
fileName should not be a parameter in this method. This is an internal detail that needs to be abstracted away from the consumer projects. This way even if the internal details change (eg. if we end up changing the logic, the consumer projects don't get affected in anyway)

Reply:
Done with the changes, I exported the path for specmatic.json from the config.js and importing it inside index.ts